packages/amplify-category-auth/src/__tests__/provider-utils/awscloudformation/utils/auth-defaults-appliers.test.ts

```diff
@@ -42,6 +42,7 @@ describe('update auth defaults applier', () => {
     } as ServiceQuestionsResult;
 
     const result = await getUpdateAuthDefaultsApplier({}, 'cognito-defaults.js', {} as ServiceQuestionsResult)(stubResult);
+    console.log(result);
```

Comment:
Do we need this here ?

Reply:
Oh no I don't I was trying to debug the test